commands/download_integration_test.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import (
+	"github.com/jfrog/jfrog-cli-core/utils/config"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func Test_Download(t *testing.T) {
+	tests := []IntegrationTest{
+		{
+			Name: "Success",
+			Function: func(t *testing.T, rtDetails *config.ArtifactoryDetails) {
+				bundleId := setUpSupportBundle(t, rtDetails)
+				bundle, err := downloadSupportBundle(30*time.Second, 100*time.Millisecond, rtDetails, bundleId)
+				require.NoError(t, err)
+				assert.Contains(t, bundle, bundleId)
+			},
+		},
+		{
+			Name: "Not found",
+			Function: func(t *testing.T, rtDetails *config.ArtifactoryDetails) {
+				bundle, err := downloadSupportBundle(1*time.Second, 100*time.Millisecond, rtDetails, "unknown")
+				require.Empty(t, bundle)
+				assert.EqualError(t, err, "http request failed with: 404 Not Found")
+			},
+		},
+	}
+	RunIntegrationTests(t, tests)
+}
+
+func setUpSupportBundle(t *testing.T, rtDetails *config.ArtifactoryDetails) string {
+	t.Helper()
+	conf := supportBundleCommandConfiguration{caseNumber: "foo"}
+	r, err := createSupportBundle(nil, rtDetails, &conf)
```

Comment:
You'll be able to remove the `nil` arg after a rebase on master.

Reply:
done